### ISSUE TITLE
0365 connectors retirement

### ISSUE THREAD

**Issue by kamronw18** [freerangeeggs/PostToOffice365Connector#28]
Hello! 

Will this task continue to work with Workflows once 0365 connectors are [retired](https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/) in Teams?

**Comment by freerangeeggs** [comment]
Hi @kamronw18,

I'm sorry to say it will not work when the retirement is rolled out.
I wouldn't recommend adding this extension if you are just seeing it for the first time and looking to move away if you are.

Microsoft suggests using the Workflows App in teams, but after an initial look there is some functionality, but I doubt you'll be able to create a message with the values of pipeline variables and whatever else was available to you while the pipeline is running.

Another issue is that these Workflow Connectors (not to be confused with the Teams Connectors that are being shut down) are premium connectors and therefore will cost money to use.

![image](https://github.com/user-attachments/assets/12c7842e-1136-403a-ad23-2692a0adb850)

In short, I'm unaware of a decent alternative, but depending on what you are after, the Teams Workflows or the [Azure Pipelines App for Teams](https://learn.microsoft.com/en-us/azure/devops/pipelines/integrations/microsoft-teams?view=azure-devops) may give you what you need.

There may be ability to build out a new Workflow Connector, similar in fashion to this extension, but given Microsoft have made the pre-existing connectors for Azure DevOps premium, there may be something stopping us from making one.

**Comment by kamronw18** [comment]
Gotcha, thank you for looking into this for me!
